filters.json.example

```diff
@@ -288,5 +288,42 @@
         "Lugia":"False",
         "Ho-Oh":"False",
         "Celebi":"False"
+    },
+    "raids":{
+        "enabled":"True",
+        "eggs": {
+            "min_level": "1",
+            "max_level": "5",
+            "ignore_eggs": "False"
+        },
+        "default": {
+            "min_dist":"0", "max_dist":"inf", "min_cp": "0", "max_cp": "999999", "min_iv":"0", "max_iv":"100",
+            "min_atk": "0", "max_atk":"15", "min_def": "0", "max_def":"15", "min_sta": "0", "max_sta":"15",
+            "quick_move": null, "charge_move": null, "moveset": null,
+            "size": null, "gender": null, "form": null, "ignore_missing": "False"
+        },
+        "Bayleef":"True",
+        "Croconaw":"True",
+        "Magikarp":"True",
+        "Quilava":"True",
+        "Electabuzz":"True",
+        "Exeggutor":"True",
+        "Magmar":"True",
+        "Muk":"True",
+        "Weezing":"True",
+        "Alakazam":"True",
+        "Arcanine":"True",
+        "Flareon":"True",
+        "Gengar":"True",
+        "Jolteon":"True",
+        "Machamp":"True",
+        "Vaporeon":"True",
+        "Blastoise":"True",
+        "Charizard":"True",
+        "Lapras":"True",
+        "Rhydon":"True",
+        "Snorlax":"True",
+        "Tyranitar":"True",
+        "Venasaur":"True"
```

Comment:
error in line 327:

> [    MainProcess][   Filters][   ERROR] Unable to find pokemon named 'Venasaur'...
[    MainProcess][   Filters][   ERROR] Please see PokeAlarm documentation for proper Filter file formatting.

replace to correct name Venusaur